index.html

```diff
@@ -12,7 +12,26 @@
       <h1>Quiz App</h1>
     </header>
     <main>
-      <section class="placeholder"></section>
+      <article class="card">
+        <svg
```

Comment:
This a one good way. Maybe it will be easier to just reference the svg with the img element?